.github/workflows/root.yml

```diff
@@ -71,14 +71,35 @@ jobs:
           name: Production JAR
           path: build/artifacts/KDBot.jar
 
-      - name: Docker Login
-        run: "docker login -u ${{ secrets.MAVEN_USER }} -p ${{ secrets.MAVEN_PASSWORD }} docker.kotlindiscord.com"
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
What's this for?

Reply:
I followed build and push action's official docs, and all examples had it.

Reply:
I'm guessing you should use it if you are targeting something else than AMD64, so we shouldn't need it

Reply:
Removed in 67d81d3